manifests/init.pp

```diff
@@ -85,8 +87,20 @@
   Stdlib::Absolutepath $config_path_v6          = $bird::params::config_path_v6,
   Optional[Stdlib::Filesource] $config_file_v6  = undef,
   Optional[String[1]] $config_template_v6       = undef,
+  Boolean $manage_repository                    = false,
```

Comment:
For consistency with other projects, maybe just `manage_repo`?

Reply:
ok